app/frontend/courses/curricula/types/CoursesCurriculum__Grading.res

```diff
@@ -17,14 +17,13 @@ let grade = t => t.grade
 
 let pending = evaluation => evaluation |> List.exists(grading => grading.grade == None)
 
-let isFail = (passGrade, grading) =>
+let isFail = grading =>
   switch grading.grade {
-  | Some(grade) => grade < passGrade
-  | None => false
+  | Some(_) => false
+  | None => true
   }
 
-let anyFail = (passGrade, evaluation) =>
-  evaluation |> List.exists(grading => grading |> isFail(passGrade))
+let anyFail = evaluation => evaluation |> List.exists(grading => false)
```

Comment:
Always fix warnings;
![image](https://github.com/pupilfirst/pupilfirst/assets/14979190/45ff24c3-f271-4511-a803-23342755ad0a)

Also, its an unused function; You can safely remove it;